src/main/scala/io/iohk/ethereum/network/p2p/messages/CommonMessages.scala

```diff
@@ -121,6 +122,13 @@ object CommonMessages {
         s"totalDifficulty: $totalDifficulty" +
         s"}"
 
+    override def toShortString: String =
+      s"NewBlock { " +
+        s"code: $code, " +
+        s"block: ${Hex.toHexString(block.hash.toArray[Byte])}, " +
```

Comment:
This log proved really useful to check full non main chain blocks on the testnet, so I don't agree with this change

With all the other changes logged in this PR I'm not sure if there's any way of getting them

Reply:
Sorry can you elaborate on what you mean by "full non main chain blocks"? Are there specific parts of the header you'd like to see? The default included the [header and the body](https://github.com/input-output-hk/mantis/blob/20369d331969884c474eb15c11900744ec1e4082/src/main/scala/io/iohk/ethereum/domain/Block.scala#L15), did you really benefit from seeing for example the [transaction payload](https://github.com/input-output-hk/mantis/blob/20369d331969884c474eb15c11900744ec1e4082/src/main/scala/io/iohk/ethereum/domain/Transaction.scala#L45) in the log? Is there a middle ground?

Reply:
Changed it so the header is included.